changelog.md

```diff
@@ -99,6 +99,7 @@ Getting the correct TSS address for Bitcoin now requires proviidng the Bitcoin c
 * enable zetaclients to use dynamic gas price on zetachain - enables >0 min_gas_price in feemarket module
 * add static chain data for Sepolia testnet
 * added metrics to track the burn rate of the hotkey in the telemetry server as well as prometheus
+* [1425](https://github.com/zeta-chain/node/pull/1425) add `whitelist-erc20` command
```

Comment:
Would need to be added above under unreleased

Reply:
Will this affect the execution of `make generate`？

Reply:
No